src/Test/AspNetCoreWebsite/Program.cs

```diff
@@ -15,20 +15,49 @@
 [assembly: DomainServiceEndpointRoutePattern(EndpointRoutePattern.FullName)]
 
 var builder = WebApplication.CreateBuilder(args);
-builder.Services.AddOpenRiaServices();
```

Comment:
TODO: determine which of the options to keep and if we want any of the commented out code samples